enterprise/frontend/src/embedding-sdk/cli/utils/ask-for-db-connection-info.ts

```diff
@@ -33,19 +33,6 @@ export async function askForDatabaseConnectionInfo(options: Options) {
       continue;
     }
 
-    const visibleIf = field["visible-if"];
-
-    const shouldShowField =
-      !visibleIf ||
-      Object.entries(visibleIf).every(
-        ([key, expected]) => connection[key] === expected,
```

Comment:
My solution to this was actually to do:

```
([key, expected]) => !!(connection[key]) === !!(expected)
```

since this comparison was basically saying that `false` and `undefined` are different values. But in actuality we just need to check if they're both falsy

Reply:
ah. I just read your other comment. Feel free to ignore this